### ISSUE TITLE
Problems with localization

### ISSUE THREAD

**Issue by Airon90** [mozillahispano/openwapp#137]
[ ] "country" variable is not localized ([see](https://www.transifex.com/projects/p/openwapp/viewstrings/#en/openwappen-usproperties/40997289), [see](https://www.transifex.com/projects/p/openwapp/viewstrings/#en/openwappen-usproperties/40997315))
[ ] "minutes" variable can be "1", so «minutes» in the key should be at least «minute(s)» or another key for the singular word should be added ([see](https://www.transifex.com/projects/p/openwapp/viewstrings/#en/openwappen-usproperties/40997290))
[ ] Why don't use just «Sign-up» instead of «OpenWapp Sign-up»? It is obvious that this is the sign-up to OpenWapp! Moreover the Italian translation of that key («Registrazione su OpenWapp») is too long. ([see](https://www.transifex.com/projects/p/openwapp/viewstrings/#en/openwappen-usproperties/40997313))

**Comment by Timmaks** [comment]
Yes, the variable part is next on the list. My first goal was to port the l10n and activate the new languages.

Feel free to  work on this

**Comment by Timmaks** [comment]
I think it is ok to drop OpenWapp as you suggested

**Comment by Airon90** [comment]
I'd help you if only I knew how to code and how to read code ._.

**Comment by Timmaks** [comment]
Manifest discription is now in transifex

**Comment by Timmaks** [comment]
Marketplace discription is now in transifex

**Comment by Timmaks** [comment]
commit b69bbf8 remove openwapp from the sign-up label

**Comment by bidaian** [comment]
Just a word of advice, for those translating in transifex. In order to see the context when translating pluralized strings, you have to choose the tab Details and read the key. Then you will be able to see if you're translating [zero], [one], [two], [few], [many] or [other].

**Comment by bidaian** [comment]
In order to solve the first point I see three solutions:

* add the [list of countries](https://raw.githubusercontent.com/mozillahispano/openwapp/master/app/scripts/countries.json) (224) as a translatable resource to each locale and let each team translate it. An additional lookup will be needed in the script. Translation is probably already done, for example it is [here for eo](https://hg.mozilla.org/releases/l10n/mozilla-aurora/eo/file/default/toolkit/chrome/global/regionNames.properties]) and [here for es-AR](https://hg.mozilla.org/releases/l10n/mozilla-aurora/es-AR/file/default/toolkit/chrome/global/regionNames.properties). Firefox uses this approach. Both the developers and the translators have to do something.

* use the native name for full_name in the [list](https://raw.githubusercontent.com/mozillahispano/openwapp/master/app/scripts/countries.json). Germany will be listed as "Deutschland" and  Belarus as "Беларусь", no matter the locale being used. That's what FirefoxOS (Gaia) does for language names, when you have to choose among them. They all appear in their native form. There is no work involved for the localizers.

* wontfix.

I'm willing to help. But since there's some work involved, I'd like to know if one of the solutions is more likely to be adopted.

If you ask me, the first one is a bit of overkill for this app, I'd go for the second one, changing full_name.

**Comment by bidaian** [comment]
Actually, it is trickier. Countries with many official languages and the like. So I prepared a pull request adding the country names as a localizable resource.
